src/programs/match_template/match_template.cpp

```diff
@@ -146,7 +160,7 @@ void MatchTemplateApp::DoInteractiveUserInput( ) {
     wxString correlation_avg_output_file;
     wxString scaled_mip_output_file;
 
-    float pixel_size              = 1.0f;
+    pixel_size                    = 1.0f;
```

Comment:
Why was the definition moved?

Reply:
It is a bit of laziness.

I wanted access to "pixel_size" inside void MatchTemplateApp::MasterHandleProgramDefinedResult but didn't want to have to modify the results array, since all those values (1-6) are basically a hard-coded header. (see variable number_of_meta_data_values)

This is for an experimental feature CISTEM_TEST_FILTERED_MIP that is useful for smaller or symmetric samples that have more correlation between rotational positions. it is a hack though.